velox/dwio/common/SelectiveStructColumnReader.cpp

```diff
@@ -78,6 +78,20 @@ void SelectiveStructColumnReaderBase::fillOutputRowsFromMutation(
   }
 }
 
+namespace {
+
+bool testFilterOnConstant(const velox::common::ScanSpec& spec) {
```

Comment:
Probably avoid method names starting with test. There are many methods for testing purpose only that starts with 'test'. It might be confusing

Reply:
Ah, I see this class has its own convention. It looks like there are many such methods. Then ignore this comment